app/controllers/visitors_controller.rb

```diff
@@ -0,0 +1,14 @@
+class VisitorsController < ApplicationController
+	before_action :authenticate_user!
+
+  def index
```

Comment:
I'm a little confused about this controller. What's a visitor?  Is this just a HomeController type thing?